feat: added pathpoint documentation (#12027)
newrelic/docs-website: src/content/docs/business-observability/install-pathpoint.mdx

---
title: Install Pathpoint
tags:
  - Business observability
  - Get started
  - Install Pathpoint
metaDescription: Learn how to install Pathpoint to your New Relic account
---

import installPathpoint from 'images/business-observability-screenshot-install-pathpoint.png'

import addPathpointApp from 'images/business-observability-screenshot-add-pathpoint-app.png'

Pathpoint can be integrated with almost any New Relic account to provide comprehensive business journey observability. To get started, it's important to have telemetry data that is relevant to your organization. This can include metrics, events, logs, or traces. 
While APM events and logs are a common starting point for Pathpoint, the choice of telemetry data is entirely up to you. 

Before you could start with Pathpoint, you need to install it to your New Relic account.

## Install Pathpoint from the New Relic catalog

To install Pathpoint:

<Steps>

<Step>

> Navigate to [New Relic](https://one.newrelic.com) and sign in with your account. 

Reviewer: ```suggestion
Log in to [New Relic](https://one.newrelic.com). 
```

</Step>

<Step>

> From left hand navigation, click the **Add data**.

Reviewer: ```suggestion
From left hand navigation, click **+ Add Data**.
```

</Step>

<Step>

> Filter the installer to look at **Apps and Visualizations**.

Reviewer: I'd remove this step.

</Step>

<Step>

> Type the word `pathpoint` in the text filter.

Reviewer: ```suggestion
Select **Apps and Visualizations**, and search for `pathpoint`.
```

</Step>

<Step>

> Click on the installation icon for the verified Pathpoint.

Reviewer: ```suggestion
Click on the installation icon for the verified **Pathpoint**, and then click **Add this app**.
```

<img
  title="Install Pathpoint"
  alt="Install Pathpoint"
  src={installPathpoint}
/>

</Step>

<Step>

> Click **Add this app**.

Reviewer: I'd remove this step.

<img
  title="Add Pathpoint to your New Relic account"
  alt="Add Pathpoint to your New Relic account"
  src={addPathpointApp}
/>

</Step>

> Pathpoint is now added to your New Relic account and can be accessed under **Apps**.

Reviewer: ```suggestion
Pathpoint is now added to your New Relic account and you can access it under **Apps**.
```

</Steps>

> Once you've installed the pathpoint, its configurations can be customized without any customization to the code, by simply upload your modified JSON file to Pathpoint. In addition you can make changes to the stages, steps, and touchpoints and save the current configuration as a JSON file. You can learn more about [configuring Pathpoint in our documentation](/docs/business-observability/configure-pathpoint/). 

Reviewer: ```suggestion
Once you've installed Pathpoint, you can customize its configuration by simply upload your modified JSON file to Pathpoint. In addition, you can make changes to the stages, steps, and touchpoints, and save the current configuration as a JSON file. You can learn more about [configuring Pathpoint in our documentation](/docs/business-observability/configure-pathpoint/). 
```
